docs/app/pages/components/components-sections/timeline/timeline/timeline.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="row">
+    <div class="col-md-10">
+        <button class="btn button-primary" (click)="addEvent()">Add New Event</button>
+        <ux-timeline>
+            <ux-timeline-event *ngFor="let event of events" [badgeColor]="event.color" [badgeTitle]="event.shortDate">
```

Comment:
Could the date pipe be used for the short date too? That way we dont need two properties on the object for date?

Reply:
The 'badgeTitle' is a required property. In this case I am using the short form of the date because that is what was used in the 'ng1' example. The test does not have to be a date.

Reply:
I just had another look at this, I think we can use the date pipe to simplify this here e.g.. 

```html
[badgeTitle]="event.shortDate | date"
```

As this is not part of the component template this will not mean this always has to be a date, but by using it here we can remove the additional `shortDate` property from each object which will simplify the example and remove the duplication.

Reply:
Removed 'shortDate' value. Now using value of 'date', along with DatePipes, to generate the two displayed dates.